mozilla/foundation.mozilla.org
# share data for blog category pages

Closes #6870


**Link to sample test page**: https://foundation-s-blog-categ-xi96db.herokuapp.com/en/blog/category/advocacy/

We have now added a new share description and image field to blog categories, that way if you update these and share the category's blog page, the metadata and preview will reflect the specifics category's description instead of the blog's as a whole. 

If the share description is not set, it will default to the category's intro text. 
If the intro text is not set, it will default to the foundation sites share description like before! 

**Steps to test:**
1. Log in with the credentials: "testuser/testuser"
2. Click "Snippets" 
3. Click "Blog Categories"
4. Select the desired category you would like to make a change to, make your desired changes and save.
5. Visit the categories page at "https://foundation-s-blog-categ-xi96db.herokuapp.com/en/blog/category/ <- Append your category's name to the end. 
6. Intro text should appear as expected.
7. Share the categories page on slack, and the desired share text and image should appear like so: 
<img width="823" alt="Screen Shot 2021-08-30 at 5 58 29 PM" src="https://user-images.githubusercontent.com/18314510/131424926-e29458fa-c697-469d-8ffd-d6026956437f.png">


## Checklist

**Changes in Models:**
